src/js/components/Select/SelectContainer.js

```diff
@@ -320,6 +320,15 @@ const SelectContainer = forwardRef(
         }
       : {};
 
+    const executeSearch = useCallback(
```

Comment:
We don't use the term "execute" in a context like this. Instead, we tend to rename the property passed in to "onSearchProp" and then call this internal callback function "onSearch".